src/tools/info/proto_info.c

```diff
@@ -245,20 +265,16 @@ print_ucp_ep_info(ucp_worker_h worker, const ucp_ep_params_t *base_ep_params,
 
     ucp_ep_print_info(ep, stdout);
 
-    status_ptr = ucp_disconnect_nb(ep);
-    tmp_status = wait_completion(worker, status_ptr);
+    tmp_status = ep_close(worker, ep, 0,
+                          (ip_addr != NULL) ? "client" :
+                                              "connected to UCP worker");
```

Comment:
can we set this string as part of the "if" in line 223

Reply:
done